lib/utils/report.mjs

```diff
@@ -170,28 +154,18 @@ function isDisabled(ruleName, startLine, disabledRanges) {
 	return false;
 }
 
-/** @param {Problem} problem */
-function isFixApplied(problem) {
-	const { end, start, index, endIndex, fix, node, result, ruleName, word } = problem;
-	const { disabledRanges, config = {}, fixersData } = result.stylelint;
+/** @param {Problem & { line: number }} problem */
+function isFixApplied({ fix, line, result: { stylelint }, ruleName }) {
+	const { disabledRanges, config = {}, fixersData } = stylelint;
 
 	if (typeof fix !== 'function') {
```

Comment:
[suggest] How about replacing `typeof` here as well? Other `typeof` in this file can be addressed, too.

```suggestion
	if (!isFn(fix)) {
```